bedrock/base/templates/includes/protocol/navigation/menu/firefox.html

```diff
@@ -0,0 +1,103 @@
+                <img src="{{ static('protocol/img/logos/firefox/firefox.png') }}" class="mzp-c-menu-item-icon" width="24" height="24" alt="">
+                <h4 class="mzp-c-menu-item-title">{{ _('Firefox Quantum Desktop Browser') }}</h4>
+                <p class="mzp-c-menu-item-desc">{{ _('Get the browser gives more power to you on Windows, Mac OS or Linux.') }}</p>
+              </a>
+              <ul class="mzp-c-menu-item-list">
+                <li><a href="{{ url('firefox.new') }}" data-link-name="Download" data-link-type="nav" data-link-position="subnav" data-link-group="firefox">{{ _('Download') }}</a></li>
+                <li><a href="{{ url('firefox') }}" data-link-name="New Features" data-link-type="nav" data-link-position="subnav" data-link-group="firefox">{{ _('New Features') }}</a></li>
+                <li><a href="https://addons.mozilla.org/firefox/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=nav&amp;utm_content=firefox" data-link-name="Extensions" data-link-type="nav" data-link-position="subnav" data-link-group="firefox">{{ _('Extensions') }}</a></li>
+                <li><a href="https://support.mozilla.org/products/firefox/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=nav&amp;utm_content=firefox" data-link-name="Support" data-link-type="nav" data-link-position="subnav" data-link-group="firefox">{{ _('Support') }}</a></li>
+                <li><a href="{{ url('privacy.notices.firefox') }}" data-link-name="Privacy" data-link-type="nav" data-link-position="subnav" data-link-group="firefox">{{ _('Privacy') }}</a></li>
+              </ul>
+            </section>
+          </li>
+          <li>
+            <section class="mzp-c-menu-item mzp-has-icon">
+              <a class="mzp-c-menu-item-link" href="{{ url('firefox.mobile') }}" data-link-name="Firefox Mobile Browsers" data-link-type="nav" data-link-position="topnav" data-link-group="firefox">
+                <svg class="mzp-c-menu-item-icon" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M7 1h10a3 3 0 0 1 3 3v16a3 3 0 0 1-3 3H7a3 3 0 0 1-3-3V4a3 3 0 0 1 3-3zm0 2a1 1 0 0 0-1 1v16a1 1 0 0 0 1 1h10a1 1 0 0 0 1-1V4a1 1 0 0 0-1-1H7zm3 16a1 1 0 0 1 0-2h4a1 1 0 0 1 0 2h-4z" fill="#000" fill-rule="nonzero"/></svg>
+                <h4 class="mzp-c-menu-item-title">{{ _('Firefox Mobile Browsers') }}</h4>
+                <p class="mzp-c-menu-item-desc">{{ _('Take Firefox and Firefox Focus with you. For Android and iOS.') }}</p>
+              </a>
+              <ul class="mzp-c-menu-item-list">
+                <li><a href="{{ url('firefox.mobile') }}" data-link-name="Get Firefox" data-link-type="nav" data-link-position="subnav" data-link-group="firefox">{{ _('Get Firefox') }}</a></li>
+                <li><a href="{{ url('firefox.mobile') }}" data-link-name="Get Firefox Focus" data-link-type="nav" data-link-position="subnav" data-link-group="firefox">{{ _('Get Firefox Focus') }}</a></li>
```

Comment:
Is it possible / desirable to link to `#focus` on this page?

Reply:
Good suggestion. As there are CTA's for both FF and Focus on the same level at the top of the page, I think I'm going to leave this as-is for now. The Focus section has more info, but I'm not sure if the CTA as quite as clear :/